packages/rdf-editor/src/RdfEditor.js

```diff
@@ -115,7 +119,7 @@ export class RdfEditor extends Editor {
       shouldSerialize = hasQuads
     }
 
-    if (shouldSerialize) {
+    if (shouldSerialize && !this.noReserialize) {
```

Comment:
Does this only apply when changing format, or will it prevent any serialization when the content is updated?

Reply:
damn, I think you're right. must add a test to cover that bc it's apparently missing